pkg/csi/service/common/constants.go

```diff
@@ -162,16 +162,24 @@ const (
 	// IPs is Client IP address, IP range or IP subnet
 	IPs string = "ips"
 
-	// CSINamespace is the namespace of pvCSI in TKC Cluster
-	CSINamespace = "vmware-system-csi"
+	// CSINamespaceVanillaK8S is the namespace of CSI driver in Vanilla K8S
+	CSINamespaceVanillaK8S = "kube-system"
```

Comment:
Different vendor may choose different namespace for the deployment of vanilla driver. Can we allow user to specify this namespace, in the conf file?

Reply:
As discussed, this will be addressed in the upcoming PRs